### PR TITLE
Support DxM solutions

### PR DESCRIPTION
**Pull request by janstaelensskyline**

### CODE REVIEW

File: working/templates/dxmsolution/InstallerProject/InstallerProject.wixproj [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    <Content Include="icon.ico" />
+    <Content Include="$DxmModuleName$.wxi" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="$DxmModuleName$.xslt" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="WixToolset.Util.wixext" Version="4.0.5" />
+    <PackageReference Include="WixToolset.UI.wixext" Version="4.0.5" />
+    <PackageReference Include="WixToolset.Heat" Version="4.0.5" />
```
Comment (MichielOda) [inline working/templates/dxmsolution/InstallerProject/InstallerProject.wixproj:28]: Version 5.0.0 has been released recently. Not sure if that one can be used already or not.

File: working/templates/dxmsolution/ServiceProject/Worker.cs
Comment (MichielOda) [inline working/templates/dxmsolution/ServiceProject/Worker.cs:1]: Indentation isn't correct.

**Review by MichielOda — CHANGES_REQUESTED** [review]

File: working/templates/dxmsolution/InstallerProject/InstallerProject.wixproj (hunk 1 above)
Comment (janstaelensskyline) [reply working/templates/dxmsolution/InstallerProject/InstallerProject.wixproj:28]: Let's wait with this. It's a Major version change from last week, breaking changes that I'm not sure apply here or not. We've done all QA with the WiX 4  so far and can guarantee a stable release with this for now.

File: working/templates/dxmsolution/ServiceProject/Worker.cs
Comment (janstaelensskyline) [reply working/templates/dxmsolution/ServiceProject/Worker.cs:1]: updated

**Review by MichielOda — APPROVED** [review]

File: working/templates/dxmsolution/InstallerProject/InstallerProject.wixproj [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+		  <HarvestDirectory Include="$(SolutionDir)$ServiceName$/bin/$(Configuration)/net8.0/runtimes/win">
+			  <SuppressRegistry>true</SuppressRegistry>
+			  <SuppressRootDirectory>true</SuppressRootDirectory>
+			  <SuppressCom>true</SuppressCom>
+			  <KeepEmptyDirectories>false</KeepEmptyDirectories>
+			  <ComponentGroupName>$DxmModuleName$DotNetWinRuntimeComponents</ComponentGroupName>
+			  <DirectoryRefId>INSTALLFOLDERRUNTIMESWIN</DirectoryRefId>
+		  </HarvestDirectory>
+		  <BindPath Include="$(SolutionDir)$ServiceName$/bin/$(Configuration)/net8.0/runtimes/win" />
+	  </ItemGroup>
```
Comment (MichielOda) [inline working/templates/dxmsolution/InstallerProject/InstallerProject.wixproj:73]: Indentation is different

File: working/templates/dxmsolution/ServiceProject/ServiceProject.csproj [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <UserSecretsId>dotnet-$ServiceName$-62852eab-befd-4bf5-96a0-b49639a2f070</UserSecretsId>
+	<AssemblyName>$FullDxmName$</AssemblyName>
+	<Version>1.0.1</Version>
+	<RootNamespace>$ServiceNameFull$</RootNamespace>
+	<ServerGarbageCollection>true</ServerGarbageCollection>
```
Comment (MichielOda) [inline working/templates/dxmsolution/ServiceProject/ServiceProject.csproj:11]: Indentation is different

File: working/templates/dxmsolution/ServiceProject/Properties/launchSettings.json
Comment (MichielOda) [inline working/templates/dxmsolution/ServiceProject/Properties/launchSettings.json:1]: Not sure if these launch settings need to be part of the template.

**Review by MichielOda — CHANGES_REQUESTED** [review]

File: working/templates/dxmsolution/ServiceProject/Properties/launchSettings.json
Comment (janstaelensskyline) [reply working/templates/dxmsolution/ServiceProject/Properties/launchSettings.json:1]: They are also part of the official microsoft template for background worker service.

File: working/templates/dxmsolution/InstallerProject/InstallerProject.wixproj (hunk 2 above)
Comment (janstaelensskyline) [reply working/templates/dxmsolution/InstallerProject/InstallerProject.wixproj]: Used notepad++ pretty print. Updated.

File: working/templates/dxmsolution/ServiceProject/ServiceProject.csproj (hunk 3 above)
Comment (janstaelensskyline) [reply working/templates/dxmsolution/ServiceProject/ServiceProject.csproj]: Used notepad++ pretty print. Updated.

**Review by MichielOda — APPROVED** [review]

**Pull request merged**